src/pubsub.ts

```diff
@@ -63,7 +68,11 @@ export interface ClientConfig extends gax.GrpcClientOptions {
   apiEndpoint?: string;
   servicePath?: string;
   port?: string | number;
-  sslCreds?: ChannelCredentials;
+  // TODO: sslCreds is supposed to be of type ChannelCredentials.
+  // Using any until google-gax changes to @grpc/grpc-js by default.
+  // tslint:disable-next-line no-any
+  sslCreds?: any;
```

Comment:
Can we open a tracking issue for this?

Reply:
Done! https://github.com/googleapis/nodejs-pubsub/issues/595